wfe/web-front-end.go

```diff
@@ -587,6 +588,14 @@ func (wfe *WebFrontEndImpl) RevokeCertificate(response http.ResponseWriter, requ
 	}
 }
 
+func (wfe *WebFrontEndImpl) logCsrPem(cr core.CertificateRequest) {
+	pem := pem.EncodeToMemory(&pem.Block{
+		Type: "CERTIFICATE REQUEST",
+		Bytes: cr.Bytes,
+	})
+	wfe.log.AuditObject("Certificate request", string(pem))
```

Comment:
`AuditObject` assumes the provided interface is a valid JSON object, it'll be Marshaled to a string fine but attempting to unmarshal it will result in an error. This is *probably* fine but for the sanity of tools we develop to check this stuff later you might want to make the object you pass a `map[string]string`.